Kudu.Contracts/Infrastructure/LockExtensions.cs

```diff
@@ -13,11 +13,12 @@ public static class LockExtensions
         // return true if lock acquired and operation executed
         public static bool TryLockOperation(this IOperationLock lockObj,
```

Comment:
Remove TryLockOperationAsync.   

I still keep TryLockOperation (sync version), since LockOperation depends on it.   Also I keep it `public` as there are various unit tests for TryLockOperation.